src/highdicom/sr/sop.py

```diff
@@ -770,3 +783,49 @@ def from_dataset(
         sop_instance = super().from_dataset(dataset, copy=copy)
         sop_instance.__class__ = Comprehensive3DSR
         return cast(Comprehensive3DSR, sop_instance)
+
+
+def srread(
+    fp: Union[str, bytes, PathLike, BinaryIO],
+) -> Union[EnhancedSR, ComprehensiveSR, Comprehensive3DSR]:
+    """Read a Structured Report (SR) document in DICOM File format.
+
+    The object is returned as an instance of the highdicom class corresponding
+    to the dataset's IOD. Currently supported IODs are:
+
+    * EnhancedSRStorage via class :class:`EnhancedSR`
+    * ComprehensiveSRStorage via class :class:`ComprehensiveSR`
+    * Comprehensive3DSRStorage via class :class:`Comprehensive3DSR`
```

Comment:
The names of the IODs are as follows:

```suggestion
    * Enhanced SR via class :class:`EnhancedSR`
    * Comprehensive SR via class :class:`ComprehensiveSR`
    * Comprehensive 3D SR via class :class:`Comprehensive3DSR`
```